samples/sensor/thermometer/README.rst

```diff
@@ -0,0 +1,29 @@
+.. _thermometer-samples:
+
+Thermometer sample
+##################
+
+Overview
+********
+
+A simple thermometer example, which writes the temperature to the console once per second.
+
+Building and Running
+********************
+
+It can be built and executed on bbc_microbit as follows:
+
+.. code-block:: console
+
+	make
```

Comment:
This doesn't seem right? Shouldn't it use `west build samples/sensor/thermometer`?

Reply:
It should use `zephyr-app-commands`

Reply:
Thanks for the review! It's should be okay with the last modification.